staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_proxy_test.go

```diff
@@ -118,7 +119,7 @@ func TestProxyHandler(t *testing.T) {
 			apiService: &apiregistration.APIService{
 				ObjectMeta: metav1.ObjectMeta{Name: "v1.foo"},
 				Spec: apiregistration.APIServiceSpec{
-					Service: &apiregistration.ServiceReference{},
+					Service: &apiregistration.ServiceReference{Port: pointer.Int32Ptr(443)},
```

Comment:
why is 443 added?

Reply:
in v1 apiservice, the port field becomes a pointer. i did that to avoid nil panic. but now i suppose just making it a nil will be more appropraite

Reply:
no, it cannot be nil, we have to set it to 443 in case nil panic